wallets/react-wallet-v2/src/utils/HelperUtil.ts

```diff
@@ -67,7 +67,7 @@ export function getWalletAddressFromParams(addresses: string[], params: any) {
   let address = ''
 
   addresses.forEach(addr => {
-    if (paramsString.includes(addr)) {
+    if (paramsString.toLowerCase().includes(addr.toLowerCase())) {
```

Comment:
@bkrem this change was required because `web3` sends the request payload in lower case. The error I described earlier in private was due to this check failing

it shouldn't cause a regression when used with the other dapps

Reply:
Lol I fixed exactly the same thing over here on Friday, came across it while trying to use the Sign v1 integration with https://dex.guru which also returns the address all lowercase: [`75df99c` (#73)](https://github.com/WalletConnect/web-examples/pull/73/commits/75df99c12c32c7ca993d53c16b848acc8b38590b)